core/src/main/resources/jenkins/split-plugins.txt

```diff
@@ -29,5 +29,8 @@ jdk-tool 2.112 1.0
 # JENKINS-55681
 jaxb 2.163 2.3.0 11
 
+# JENKINS-64107
+sshd 2.265 3.0
```

Comment:
Needs to be updated to reflect actual split version.

BTW @timja I should have mentioned in the PR to create the plugin that it would be appropriate to set the `jenkins.version` to an incremental version _after_ the last weekly, so it is not offered to anyone on the UC until the weekly with _this_ change is released.

Reply:
right, easy to do quickly